supertokens_python/framework/django/django_middleware.py

```diff
@@ -0,0 +1,48 @@
+import asyncio
+
+from asgiref.sync import async_to_sync
+from werkzeug import Response
+
+from supertokens_python.framework.django.django_request import DjangoRequest
+from supertokens_python.framework.django.django_response import DjangoResponse
+from supertokens_python import Supertokens
+from supertokens_python.exceptions import SuperTokensError
+from supertokens_python.session import Session
+from supertokens_python.supertokens import manage_cookies_post_response
+
+
+def middleware(get_response):
```

Comment:
what about error handler in django?

Reply:
I did not solve that problem. Lets speak tomorrow about this.